### PR TITLE
Bug 1200321 - Create Imagecompare RTL test for Settings:Network

### PR DESCRIPTION
**Pull request by nojunpark**

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=7292792c944fce6d4b2ff466c19ad77b170c3c77

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=174df8c7a1c460f9bf98fd87dd3903d4ba544f02

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=fa4950073ff6d311cd27447be8643cd52e1fd890

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=24ea6a72f5817da62e78aa00da5d307faf4e297e

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=4cf6da5b9cd1e12bcc0943bce5159fccd6025c7e

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=6ac87c89847d38d7dcd370a7caaef830d82fafd1

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d0d1a666f50b9a58396ac3fb9bf959bdad288510

File: tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/test_settings_NC_RTL.py [hunk 1]
```diff
@@ -51,5 +220,7 @@ def test_settings_app(self):
         hotspot_page.tap_security_settings()
         self.take_screenshot('internet_sharing-hotspot-security')
         hotspot_page.confirm_security_settings()
-        settings.return_to_prev_menu(internet_settings_page.screen_element)
-        settings.return_to_prev_menu(settings.screen_element)
+        settings.return_to_prev_menu(internet_settings_page.screen_element,
+                                     hotspot_page.screen_element)
+        settings.return_to_prev_menu(settings.screen_element,
+                                     internet_settings_page.screen_element)
```
Comment (mwargers) [inline tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/test_settings_NC_RTL.py]: Oh, after running this test, my data connection was enabled.

You might want to add a tearDown function where you disable this again, like here:
http://mxr.mozilla.org/gaia/source/tests/python/gaia-ui-tests/gaiatest/tests/functional/cost_control/test_cost_control_data_alert_mobile.py#58

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=af19d13724ee04f7deb490903bf4264bd93ac00d

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=a2d9461c9d670be00cc2b102448373a58830ec60

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=292b48002bd92b2609a8491eb814f25fdb3fafbd

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=7a0ebe3d2ffac3162417b92c57c272286b2b5673

**Pull request merged**